db-checkbook/build_scripts/01_dataloading.py

```diff
@@ -1,29 +1,62 @@
-from dcpy.connectors import s3
 import pandas as pd
+import geopandas as gpd
+import boto3
+import os
+from pathlib import Path
+from dotenv import load_dotenv
 
 BASE_BUCKET = 'edm-recipes'
 BASE_URL = "https://edm-recipes.nyc3.cdn.digitaloceanspaces.com"
 
+_curr_file_path = Path(__file__).resolve()
+LIB_DIR = _curr_file_path.parent.parent / '.library'
+load_dotenv(_curr_file_path.parent.parent.parent / '.env') 
 
 
-def read_s3_edm_recipes_cpdb(version, type_geom, save_file_path):
+
+def download_s3_edm_recipes_cpdb():
     """read EDM data: using S3 connectors
     example: datasets/dcp_cpdb/2018_adopted_polygons/
     version: 2018_adopted, 2019_adopted, 2020_adopted, 2021_adopted, 2022_adopted, 2023_executive
     type_geom: _polygons, _points 
     """
-    digital_ocean_filepath = f'datasets/dcp_cpdb/{version}{type_geom}/dcp_cpdb.shp'
-    s3.client().download_file(BASE_BUCKET, digital_ocean_filepath, save_file_path)
-    return save_file_path
+
+    s3_resource = boto3.resource('s3', 
+        aws_access_key_id=os.environ["AWS_ACCESS_KEY_ID"],
+        aws_secret_access_key=os.environ["AWS_SECRET_ACCESS_KEY"],
+        endpoint_url=os.environ["AWS_S3_ENDPOINT"] 
+        )
+
+    prefix = 'datasets/dcp_cpdb/'
+
+    bucket = s3_resource.Bucket(BASE_BUCKET)     
+    for obj in bucket.objects.filter(Prefix = prefix):
+        key = obj.key.replace(prefix, '')
+        if not os.path.exists(LIB_DIR / os.path.dirname(key)):
+            os.makedirs(LIB_DIR / os.path.dirname(key))
+        bucket.download_file(obj.key, LIB_DIR / key)
+
+    return ''
```

Comment:
Very minor, but I would just `return` rather than return an empty string

Reply:
maybe an enhancement for later, but a generalized version of this might be nice for `dcpy`